data/jsons/dependencies.json

```diff
@@ -0,0 +1,123 @@
+{
+    "dependencies": [
+	{
+	    "dependency": "check",
+	    "type": "pkg-config",
+	    "pkgname": "check"
```

Comment:
Having `atleast-version`, `max-version`, and `exact-version` keys for `pkg-config` types is important, as we often use features available in latest or close-to-latest versions of libraries.